pages/getting-started/quick-start/generate.mdx

```diff
@@ -70,203 +52,188 @@ The diagram above represents the relationships of your database.
+You can read it as "comments times 3".
 
 ```ts snaplet.config.ts
-/// <reference path=".snaplet/snaplet.d.ts" />
-import { defineConfig } from 'snaplet';
-import { copycat } from '@snaplet/copycat';
-
-export default defineConfig({
-  generate: {
-    async run(snaplet) {
-      await snaplet.posts([{
-        title: 'Hello World!',
-        // focus(1:6)
-        author: {
-          email: ({ seed }) =>
-            copycat.email(seed, {
-              domain: 'acme.org'
-            }),
-        },
-      }]);
+import { SnapletClient } from '@snaplet/seed';
+
+const snaplet = new SnapletClient();
+
+await snaplet.posts([
+  {
+    title: 'Hello World!',
+    author: {
+      email: ({ seed }) =>
+        copycat.email(seed, {
+          domain: 'acme.org',
+        }),
     },
+    // focus
+    comments: (x) => x(3),
   },
-});
+]);
 ```
 
 ---
 
-Finally, we also define the comments of the post represented by the `comments` field.
-As we need more than one comment, we can use the `x` function which will create a list of similar comments.
-You can read it as "comments times 3".
+Finally, we'll add a `dryRun` option when instantiating `SnapletClient`. When we run our script with a `DRY` environment variable set, we'll log the sql statements
+to the console instead altering the database.
 
 ```ts snaplet.config.ts
-/// <reference path=".snaplet/snaplet.d.ts" />
-import { defineConfig } from 'snaplet';
-import { copycat } from '@snaplet/copycat';
-
-export default defineConfig({
-  generate: {
-    async run(snaplet) {
-      await snaplet.posts([{
-        title: 'Hello World!',
-        author: {
-          email: ({ seed }) =>
-            copycat.email(seed, {
-              domain: 'acme.org'
-            }),
-        },
-        // focus
-        comments: (x) => x(3),
-      }]);
+import { SnapletClient } from '@snaplet/seed';
+
+const snaplet = new SnapletClient({
+  // focus
+  dryRun: Boolean(+process.env.DRY)
+});
+
+await snaplet.posts([
+  {
+    title: 'Hello World!',
+    author: {
+      email: ({ seed }) =>
+        copycat.email(seed, {
+          domain: 'acme.org',
+        }),
     },
+    comments: (x) => x(3),
   },
-});
+]);
 ```
 
 ---
 
 We're done!
 
-Copy the code and add it to your `snaplet.config.ts` file.
+Copy the code and add it to your `seed.mts` file.
 
 ```ts snaplet.config.ts
-/// <reference path=".snaplet/snaplet.d.ts" />
-import { defineConfig } from 'snaplet';
-import { copycat } from '@snaplet/copycat';
-
-export default defineConfig({
-  generate: {
-    async run(snaplet) {
-      await snaplet.posts([{
-        title: 'Hello World!',
-        author: {
-          email: ({ seed }) =>
-            copycat.email(seed, {
-              domain: 'acme.org'
-            }),
-        },
-        comments: (x) => x(3),
-      }]);
+import { SnapletClient } from '@snaplet/seed';
+
+const snaplet = new SnapletClient({
+  dryRun: Boolean(+process.env.DRY)
```

Comment:
Could we maybe go with `process.env.DRY !== undefined` so there is no "magic" casting, as it's an example file I think simpler is better.

Reply:
Sure, I agree. I went with `process.env.DRY === '1'` in the`seed.mts` example script we generate, I'll do that here too. That ok?

Reply:
Yes perfect!